external/LibraryThemeStyles/src/Controller/AdminController.php

```
@@ -0,0 +1,184 @@
+        } catch (\Throwable $e) {
+            $error = 'Error: ' . $e->getMessage();
+        }
+
+        return new ViewModel([
+            'message' => $message,
+            'error' => $error,
+            'siteSlug' => $siteSlug,
+        ]);
+    }
+
+    private function applyPresetToThemeSettings(?string $siteSlug, string $themeKey, string $preset): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$themeKey'. (undefined)

(UnusedFormalParameter)